cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -415,39 +413,38 @@ class TemporalTest extends DocumentingTest {
       section("Specifying durations", "cypher-temporal-specifying-durations") {
         p("""A _Duration_ represents a temporal amount, capturing the difference in time between two instants, and can be negative.""")
         p(
-          """A _Duration_ may be specified using either a _unit based form_ or a _date-and-time based form_:
+          """The specification of a _Duration_ is prefixed with a **`P`**, and can use either a _unit-based form_ or a _date-and-time-based form_:
             |
-            |* Unit based form: **`P`**`[n`**`Y`**`][n`**`M`**`][n`**`W`**`][n`**`D`**`][`**`T`**`[n`**`H`**`][n`**`M`**`][n`**`S`**`]]`
+            |* Unit-based form: **`P`**`[n`**`Y`**`][n`**`M`**`][n`**`W`**`][n`**`D`**`][`**`T`**`[n`**`H`**`][n`**`M`**`][n`**`S`**`]]`
             | ** The square brackets (`[]`) denote an optional component (components with a zero value may be omitted).
             | ** The `n` denotes a numeric value which can be arbitrarily large.
-            | ** The value of the last -- and least significant -- component may contain a decimal fraction.
+            | ** The value of the last, and least significant, component may contain a decimal fraction.
```

Comment:
This has changed the meaning of the sentence entirely (similarly to the offsets in an earlier comment): from a qualification, we now have a conjunction, which is incorrect. Please revert, or use brackets / i.e. or something like that.